docs/outgoing-webhooks.md

```diff
@@ -0,0 +1,114 @@
+# Outgoing Webhooks
+
+Outgoing webhooks are a type of integration where the Zulip server sends HTTP POST requests to a third party URL.
```

Comment:
I'm assuming this is *currently* POST requests, but need it be limited to this?

Reply:
Outgoing webhooks are meant for POST requests only majorly (even Slack, Mattermost supports only POST requests).

Reply:
This is just a tiny point, I just wondered if it need be limited to `POST`, eg. if a different interface was written, and whether it could more broadly be "sends HTTP requests to a third party URL."